examples/mnist_00.md

```diff
@@ -0,0 +1,242 @@
+# MNIST Basic Tutorial
+
+This tutorial series is built around the [PyTorch MNIST example] and is meant to demonstrate how to modify your PyTorch code to be configured by Hydra. We will start with the simplest case which introduces one central concept while minimizing altered code. In the following tutorials ([Intermediate][Intermediate Tutorial] and [Advanced][Advanced Tutorial]), we will show how a few additional changes can yield a very powerful end product.
+
+The source file can be found in [mnist_00.py]
+
+### The 'HYDRA BLOCK'
+***
+For clarity in this tutorial, as we modify the [PyTorch MNIST example], will make the diffs explicit. Most of the changes we introduce will be at the top of the file within the commented `##### HYDRA BLOCK #####`, though in practice much of this block could reside in its own concise imported file.
+
+### Imports
+```python
+import hydra
+from hydra.core.config_store import ConfigStore
+from typing import List, Any
+from omegaconf import 
+from dataclasses import dataclass
+
+# config schema imports
+from config.torch.optim import AdadeltaConf
+from config.torch.optim.lr_scheduler import StepLRConf
+```
+
+There are two areas in our Hydra-specific imports. First, since we define configs in this file, we need access to the following:
+- the `ConfigStore`
+- typing from both `typing` and `omegaconf`
+- the `dataclass` decorator
+
+##### Config Store
+*where we store our configs*
+
+Briefly, the concept behind the `ConfigStore` is to create a singleton object of this class and register all config objects to it. This tutorial demonstrates the simplest approach to using the `ConfigStore`.

```

Comment:
I guess since we're committing to the only the dataclass bases style, we should remove the term "config schema" from the tutorial? We aren't really using the dataclasses as config schema. I suppose the proper term would be  "structured config templates"? @omry

Reply:
I think a more advanced version of the tutorial can get into dataclasses as schema for config files.
This in general will be much more elegant once Hydra has support for recursive default lists so we should probably wait for Hydra 1.1 for it.

Reply:
I like that plan.

I'm going to remove a little more content from the Basic example to streamline it a bit more and migrate some to the WIP Intermediate.